repl/appengine/web/src/app/repl-console/repl-console.component.html

```diff
@@ -0,0 +1,38 @@
+<div class="repl-console-wrapper" >
+<h1>REPL Console</h1>
+<p>Add statements to evaluate.</p>
+<p>Do not include sensitive data.</p>
+
+<div class="repl-input-block"
+    (keydown)="onEnter($event)">
+    <div class="statement-block" *ngFor="let response of lastEvaluate.responses; let i = index">
```

Comment:
Consider adding some margins to top and bottom around the repeated statement blocks to visually clarify which statements, errors and outputs are grouped together